web/src/components/drawer/drawer.tsx

```diff
@@ -0,0 +1,406 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk';
```

Comment:
nit: can we name the file "netflow-traffic-drawer.tsx" so it matches the component name?

Reply:
done in https://github.com/netobserv/network-observability-console-plugin/pull/551/commits/1a3e846440b784fc3a4aa8fdb3e0518a03887b88